doc/articles/features/working-with-xaml-hot-reload.md

```diff
@@ -259,39 +260,53 @@ Skia-based targets provide support for full XAML Hot Reload and C# Hot Reload. T
 - As of VS 2022 17.9 XAML or C# Hot Reload under WSL is not supported
 - VS Code
   - With the debugger: The C# Dev Kit is handling hot reload [when enabled](https://code.visualstudio.com/docs/csharp/debugging#_hot-reload). As of December 20th, 2023, C# Dev Kit hot reload does not handle class libraries. To experience the best hot reload, do not use the debugger.
-  - Without the debugger: The VS Code Uno Platform extension is handling Hot Reload (C# or XAML)
-  - Adding new C# or XAML files to a project is not yet supported
+  - Without the debugger: The VS Code Uno Platform extension is handling Hot Reload (C# or XAML).
+  - Adding new C# or XAML files to a project is not yet supported.
 - Rider
   - Hot Reload is only supported without the debugger.
-  - Adding new C# or XAML files to a project is not yet supported
+  - Adding new C# or XAML files to a project is not yet supported.
 
 ### [**WebAssembly**](#tab/wasm)
 
-WebAssembly is currently providing both full and partial Hot Reload support, depending on the IDE.
+WebAssembly is currently providing full Hot Reload support.
 
 - In Visual Studio Code:
-  - Both C# and XAML Hot Reload are fully supported
-  - Adding new C# or XAML files to the project is not yet supported
-  - Hot Reload is not supported when using the debugger
+  - Both C# and XAML Hot Reload are fully supported.
+  - Adding new C# or XAML files to the project is not yet supported.
+  - Hot Reload is not supported when using the debugger.
 - In Rider:
-  - Both C# and XAML Hot Reload are fully supported
-  - Adding new C# or XAML files to the project is not yet supported
-  - Hot Reload is not supported when using the debugger
-- In Visual Studio for Windows:
-  - [`MetadataUpdateHandlers`](https://learn.microsoft.com/dotnet/api/system.reflection.metadata.metadataupdatehandlerattribute) are invoked without the list of changed types, which means that some hot reload features may not be available. This feature is slated to be available for .NET 9.
+  - Both C# and XAML Hot Reload are fully supported.
+  - Adding new C# or XAML files to the project is not yet supported.
+  - Hot Reload is not supported when using the debugger.
+
+### [**iOS, Android**](#tab/mobile)
+
+Mobile targets now support both XAML and C# Hot Reload. Debugger-specific variations apply depending on the IDE.
+
+- In Visual Studio:
+  - The "Hot Reload on File Save" feature must be disabled to avoid crashing the app. You can find this feature by clicking on the down arrow next to the red flame button in the Visual Studio toolbar.
```

Comment:
Never use this ... but not aware it could crash the app. Is this something you experienced?

Reply:
@jeromelaban is the "Hot Reload on File Save" still something that should be disabled in some specific cases?

Reply:
This is not happening anymore.